### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_MigratetoAzureSQL_Databasebackup-networkshare_Keyboard:There is no visible indication of mark provided for password field that it is a required field.

### ISSUE THREAD

**Issue by tripathiakhil429** [microsoft/azuredatastudio#15695]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yWCAG2.1;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG3.3.1;#AINotInScope;#Keyboard

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till Migrate to azure sql and press enter.
5. Add azure account and tab till next button and press enter.
6. Select SQL server on azure virtual machine radio button and fill all the required fields and tab till next button and press enter.
7. Check Online migration radio button and tab till next button and press enter.
8. Check "My database are on a network share" radio button.
9. Tab till Password textbox.

### Actual:
There is no visible indication of mark provided for password field that it is a required field.
**This issue is observed with all the required  fields.**

### Expected:
There should be a asterix mark provided for the required fields.

### User Impact:
Keyboard only users while tabbing through the password textbox will not know that it is a required field.

**Comment by tripathiakhil429** [comment]
closing as duplicate.

**Comment by 12-shweta** [comment]
**Github Tags:**#Closed;
